modules/i3s/test/test-utils/load-utils.js

```diff
@@ -154,8 +154,13 @@ export const TILESET_STUB = () => ({
   loader: I3SLoader
 });
 
-export async function loadI3STile(options = {}) {
+export async function loadI3STile(options = {}, replaceWithKTX2Texture = false) {
   const i3sTilesetData = TILESET_STUB();
+
+  // Flag for testing purposes
+  if (replaceWithKTX2Texture) {
```

Comment:
I actually don't understand what this does from looking at the diff. Maybe explain a bit more what this does. Also if this flag is just for testing maybe add an underscore?
```suggestion
  if (_replaceWithKTX2Texture) {
```

Reply:
Yes, I see. Left a comment and renamed with underscore. Thank you!